internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -493,7 +461,12 @@ func (r *AgentServiceConfigReconciler) newAssistedServiceDeployment(instance *ad
 				MountPath: "/data",
 			},
 		},
-		Resources: corev1.ResourceRequirements{},
+		Resources: corev1.ResourceRequirements{
+			Requests: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("300m"),
+				corev1.ResourceMemory: resource.MustParse("400Mi"),
+			},
```

Comment:
```suggestion
			Requests: corev1.ResourceList{
				corev1.ResourceCPU:    resource.MustParse("200m"),
				corev1.ResourceMemory: resource.MustParse("512Mi"),
			},
```

These request values seem to be a bit low. Here's what I see from `kubectl top pod` in my test env:

```
[dev@edge-10 dev-scripts]$ kubectl top pod
NAME                               CPU(cores)   MEMORY(bytes)
assisted-service-94f447959-8fdgq   15m          517Mi
ocp-metal-ui-d5df44bd5-2s8v8       0m           5Mi
postgres-7ccd794b8-dbdrv           30m          104Mi
```

AI was idle when I ran this command and there were 2 cluster records. 

P.S: One could argue that using `512Mi` when the service is idle sounds like there may be a leak or some aggressive, `in-memory` caching.

Reply:
updated the requests...but I'm not looking into the possibility of a memory leak